docs/application/tizen-studio/rt-ide/overview.md

```diff
@@ -1,15 +1,31 @@
 # TizenRT
 
-Tizen platform is commercialized on smart TVs, smartphones, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of TizenRT is to extend the Tizen platform device coverage to these kind of low-end devices.
+Tizen platform is commercialized on smart TVs, smartphones, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of TizenRT, where RT stands for real time is to extend the Tizen platform device coverage to these kind of low-end IoT devices.
 
-TizenRT is an RTOS-based lightweight platform. The following are the hardware and environmental requirements:
+TizenRT is an RTOS-based lightweight platform that can be installed into these low-end IoT devices. TizenRT follows Linux-style development that helps you to easily build your business logic on top of TizenRT. Therefore, TizenRT is only for Ubuntu. 
```

Comment:
Information Required: Please provide some more information here and verify dependencies such as Ubuntu.